FWCore/ParameterSet/interface/PluginDescription.h

```diff
@@ -124,7 +124,7 @@ namespace edm {
 
     void validate_(ParameterSet& pset, std::set<std::string>& validatedLabels, bool optional) const final {
       loadPlugin(findType(pset));
-      cache_->validate(pset);
+      parameterSetDescription_->validate(pset);
```

Comment:
The `parameterSetDescription_` appears to be used only right after `loadPlugin()`. How about `loadPlugin()` would return the `ParameterSetDescription` object for which the `validate()` (and `writeCfi()` below) are called?

Or is there some other need to keep the object alive longer?

Reply:
I agree with Matti.

Reply:
I agree also. That is even better. I made the change and am running the unit tests. I will push it as soon as they complete. Thanks.